extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -253,9 +253,14 @@ export class ResourceTypePickerDialog extends DialogBase {
 				}
 				this._toolsLoadingComponent.loading = false;
 			});
+			this._tools = tools;
 		}
 	}
 
+	private localizedToolStatus(status: ToolStatus): string {
+		return localize('deploymentDialog.ToolStatus.' + status, '{0}', status);
+	}
```

Comment:
This is doing nothing - the string {0} isn't able to be localized. You should be creating actual localized constants for each of the status' and using that instead.

Reply:
fixed